Four Brains/Model/DrumSounds.swift

```diff
@@ -103,29 +116,31 @@ class DrumSounds {
         for beatCardCounter in Range(0...3) {
             for note in Range(0...3) {
                 if beatCardCounter == 0 && note == 0{
-                    hiHatNoteSequence[0] = wholeBeat.hiHatPattern[beatCardCounter].beatCardNoteSequence[note]
+                    hiHatNoteSequence[0] = wholeBeat.getHiHatPattern()[beatCardCounter].getBeatCardNoteSequence()[note]
                 } else {
-                    hiHatNoteSequence.append(wholeBeat.hiHatPattern[beatCardCounter].beatCardNoteSequence[note])
+                    hiHatNoteSequence.append(wholeBeat.getHiHatPattern()[beatCardCounter].getBeatCardNoteSequence()[note])
                 }
             }
         }
         print("Hi-Hat note sequence: \(hiHatNoteSequence)")
         
-        print("parsenNoteSequence() called")
-        
-        assignDrumSounds()
+        print("parseNoteSequence() called")
         
     }
     
     func assignDrumSounds() {
-        sequencer.clearRange(start: AKDuration(beats: 0), duration: AKDuration(beats: 100))
+        sequencer.clearRange(start: Duration(beats: 0), duration: Duration(beats: 100))
         
         //MARK: Ride cymbal note assignment
-        if !mute.rideMuteState{
+        if !mute.getRideMuteState(){
             for note in Range(0...15) {
                 let position = ((Double(note) + 1.0)/4.0) - 0.25
                 if rideNoteSequence[note] == 1 {
-                    sequencer.tracks[0].add(noteNumber: 34, velocity: 200, position: AKDuration(beats: position), duration: AKDuration(beats: 1.0))
+                    sequencer.tracks[0].add(
+                        noteNumber: 34,
+                        velocity: 200,
+                        position: Duration(beats: position),
+                        duration: Duration(beats: 0.25))
```

Comment:
This is a lot better from a readability perspective. Is there such thing as a linter in swift?

Reply:
Thanks!

Reply:
After this comment, I installed a linter called swiftlint. It's working well.